fdbserver/Status.actor.cpp

```diff
@@ -669,6 +669,46 @@ struct RolesInfo {
 			if (commitBatchingWindowSize.size()) {
 				obj["commit_batching_window_size"] = addLatencyStatistics(commitBatchingWindowSize);
 			}
+
+			TraceEventFields const& commitBatchTransactions = metrics.at("CommitBatchTransactions");
+			if (commitBatchTransactions.size()) {
+				obj["commit_batch_transactions"] = addLatencyStatistics(commitBatchTransactions);
```

Comment:
Since this modified status json, you need to update `documentation/sphinx/source/mr-status-json-schemas.rst.inc` and `fdbclient/Schemas.cpp` as well.

Reply:
I added new elements to `fdbclient/Schemas.cpp`, but now there is no cluster.processes.<process>.roles objetcs in `documentation/sphinx/source/mr-status-json-schemas.rst.inc` at all.

I can add the description of the new metrics, but I cannot describe all other elements of `roles`.